mesonbuild/backend/vs2010backend.py

```diff
@@ -2116,3 +2121,23 @@ def get_gen_manifest(self, target):
             if arg == '/MANIFEST' or arg.startswith('/MANIFEST:'):
                 break
         return True
+
+    # Returns if a target embeds a manifest or not.
+    def get_gen_embedmanifest(self, target):
```

Comment:
I don't like the fact this duplicates the `get_gen_manifest` function. I think you should rather modify `get_gen_manifest`  to return a tri-state value (e.g. True, False, 'embed').

Reply:
That definitely results in a smaller diff.